src/services/Helper.js

```diff
@@ -98,3 +98,11 @@ export const avgOrFallback = (a, b, fallback = 0) => {
 
   return a / b;
 };
+
+export const avgOrFirstParameter = (a, b) => {
+  if (b === 0 || !b) {
```

Comment:
Why check !b? I don't think it can be falsy and not 0. And if it could be undefined we should use a default parameter instead (a, b = 0)